src/System.Private.TypeLoader/src/System.Private.TypeLoader.csproj

```diff
@@ -59,7 +59,11 @@
     <Compile Include="..\..\Common\src\TypeSystem\CodeGen\TypeDesc.CodeGen.cs" />
     <Compile Include="..\..\Common\src\TypeSystem\CodeGen\FieldDesc.CodeGen.cs" />
     <Compile Include="..\..\Common\src\TypeSystem\Ecma\EcmaField.CodeGen.cs" />
+    <Compile Include="..\..\Common\src\TypeSystem\Ecma\EcmaField.Serialization.cs" />
```

Comment:
These should not be needed for the runtime type system.

Reply:
I only put them in the compilation one, but yeah, not needed.